.changeset/large-feet-wash.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-techdocs-backend': minor
+'example-backend-next': minor
```

Comment:
```suggestion
```

We don't need a changeset for this package as it's not published.

Reply:
Gotcha